rdmd/config.d

```diff
@@ -0,0 +1,55 @@
+/**
+Provides hardcoded configuration settings for rdmd derived from
+the target environment
+*/
+module rdmd.config;
+
+/**
+'Namespace' struct to encapsulate all configuration settings
+fixed at compile time.  These may be hardcoded, or dependent
+on the compiler or OS.
+*/
+struct RDMDConfig
```

Comment:
Maybe just `Config`?

Reply:
Argh, we have too many `Config` declarations in Phobos already (std.getopt and std.process)

Reply:
Right, `Config` alone creates too many symbol clashes.  Having used `RDMDConfig` effectively as a placeholder for my first drafts, I actually came to the conclusion there were good reasons to have names like this (I'll explain more w.r.t. the `RDMDGlobalArgs` struct).